Makefile

```diff
@@ -1028,6 +1035,7 @@ ceph: namespace ## deploy the Ceph Pod
 	bash scripts/gen-ceph-kustomize.sh "build"
 	oc kustomize ${DEPLOY_DIR} | oc apply -f -
 	bash scripts/gen-ceph-kustomize.sh "isready"
+	bash scripts/gen-ceph-kustomize.sh "cephfs"
```

Comment:
@fultonj a new function creating a volume (called "cephfs") for manila here

Reply:
The scripting changes look reasonable to me.

Do you want to update a README with info on how to use the manila operator?